libevdev.rb

```diff
@@ -1,15 +1,16 @@
 class Libevdev < Formula
   desc "Wrapper library for evdev devices"
   homepage "http://www.freedesktop.org"
+  url    "https://www.freedesktop.org/software/libevdev/libevdev-1.5.2.tar.xz"
+  sha256 "5ee2163656a61f5703cb5c08a05c9471ffb7b640bfbe2c55194ea50d908f629b"
+  # tag "linuxbrew"
+
   bottle do
     cellar :any_skip_relocation
+    revision 2
```

Comment:
It shouldn't be necessary to modify the bottle revision (I think).

I'm actually a little confused as to why a revision change is necessary at all, since you're also doing a version upgrade in this PR.